packages/apps/stock/src/screens/internalMoves/InternalMoveLineDetailsScreen.js

```diff
@@ -67,8 +89,8 @@ const InternalMoveLineDetailsScreen = ({navigation, route}) => {
     ) {
       dispatch(
         fetchProductIndicators({
-          version: route.params.internalMoveLine.product.$version,
-          productId: route.params.internalMoveLine.product.id,
+          version: route.params.internalMoveLine.product?.$version,
+          productId: route.params.internalMoveLine.product?.id,
```

Comment:
This will not worked anymore as you are asking for internalMoveLineId in props, please adapt code